### DISCUSSION TITLE
422(Unprocessable Content)

### DISCUSSION THREAD

**Discussion by yerassyl02-y** [fastapi/fastapi#4538] [Questions] [answered]
Hi there
  showPopup() {
      if (!this.$refs.form.validate()) return;
      else {
        this.popup = true;
        this.$axios
          .post("save-request")
          .then(() => {
            this.$refs.form.reset();
          })
          .catch((err) => {
            console.error(err);
          });
      }
    },
Here I wrote the post request method, and for the first it worked well, I mean i recieved the data from the form, but now it doesn't work properly and I do not make a changes. What's the problem. Thank for all in advance!

**Comment by upeshphable** [comment, marked as answer]
You should give some example response you are getting along with what you API implementation is. FastAPI along with Pydantic, gives very descriptive error information when it comes to 422.

**Reply by yerassyl02-y** [reply]
Thanks a lot
I solved the problem
